x/ccv/provider/keeper/hooks.go

```diff
@@ -100,19 +104,6 @@ func (h Hooks) AfterUnbondingInitiated(ctx sdk.Context, id uint64) error {
 		// Do not put the unbonding op on hold if there are no consumer chains
 		return nil
 	}
-	// Call back into staking to tell it to stop this op from unbonding when the unbonding period is complete
-	if err := h.k.stakingKeeper.PutUnbondingOnHold(ctx, id); err != nil {
```

Comment:
Isn't this removal part of the removing the `VSCMaturedPacket`s work? If so, I wouldn't have expected it in this PR (?)